buildlib/az-io_demo.sh

```diff
@@ -33,7 +34,7 @@ if [ "x$server_ip" = "x" ]; then
     if ! "${workspace}/../test/apps/iodemo/io_demo" |& add_timestamp >server.log & then
         error "Failed to start server"
     fi
-    server_pid=$(pgrep -u "$USER" io_demo | awk '{print $2}')
+    server_pid=$(pgrep -u "$USER" -f iodemo)
```

Comment:
why need '-f' ? i think it could find unrelated process

Reply:
Couldn't find without `-f`, also had to use dir name, not a binary to see only one process and not parenting bash script.

Reply:
can we make a more specific pattern?